.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
-For plugin updates, please indicate which repos this should be built into:
+If your package needs to be released within one or more release streams, and/or distributions, please open PRs to each of those branches respectively. The easiest way to do this is to make the initial commit for the mainline branch (e.g. rpm/develop or deb/develop) and then cherry pick the commit hash onto each subsequent branch.
 
-* [x] Nightly
```

Comment:
I do think listing the supported branches is a good thing. Otherwise we can get PRs for unsupported branches which we'll need to reject.